test/WebJobs.Script.Tests/ScriptHostIdProviderTests.cs

```diff
@@ -37,21 +37,22 @@ public async Task GetHostIdAsync_WithConfigurationHostId_ReturnsConfigurationHos
         }
 
         [Theory]
-        [InlineData("TEST-FUNCTIONS--", "test-functions")]
-        [InlineData("TEST-FUNCTIONS-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", "test-functions-xxxxxxxxxxxxxxxxx")]
-        [InlineData("TEST-FUNCTIONS-XXXXXXXXXXXXXXXX-XXXX", "test-functions-xxxxxxxxxxxxxxxx")] /* 32nd character is a '-' */
-        [InlineData(null, null)]
-        public void GetDefaultHostId_AzureHost_ReturnsExpectedResult(string siteName, string expected)
+        [InlineData("TEST-FUNCTIONS--", "123123", "test-functions")]
+        [InlineData("TEST-FUNCTIONS-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", "123123", "test-functions-xxxxxxxxxxxxxxxxx")]
+        [InlineData("TEST-FUNCTIONS-XXXXXXXXXXXXXXXX-XXXX", "123123", "test-functions-xxxxxxxxxxxxxxxx")] /* 32nd character is a '-' */
+        [InlineData("TEST-FUNCTIONS-XXXXXXXXXXXXXXXX-XXXX", null, "f6c34226-637100850484032865")]
+        [InlineData(null, "123123", null)]
```

Comment:
Nit: Add separate test for Linux that will help highlight difference between hosting environments

Reply:
Indeed. Is this test exercising your new code path? It would be good to have validation that your container name is used.

Reply:
yes it tests that if in Linux consumption environment, the host id is derived from the "CONTAINER_NAME" variable.

Reply:
Ah... wasn't clear to me that `IsLinuxConsumption` was only checking the instance ID on top of the container name.

That looks good

Reply:
No problem. I added a comment to that testcase.